lang/translation.json

```diff
@@ -73,23 +73,23 @@
 	  },
 	  {
 	    "tag" : "word_maximum_value",
-	    "en" : "Filter on the maximal value",
+	    "en" : "Filter on the maximum value",
 	    "nl" : "Filteren op maximale waarde"},
 	    {
 	      "tag" : "tool_maximum_value_1_expl",
-	      "en" : "",
+	      "en" : "Every measurment reported by the sensor can provide information about the air quality at its location. However, if the sensor is dirty or damaged, then the measurment may not be representative of the air quality. This will often result in high measurment, for example 300 μg/m³. For the analysis you want to estimate the air quality, without distortions due to such outliers. To that end you can use a filter to remove measurements above a maximum value. By default, this filter is set to 10% percentile of the sensor measurements. You can adjust this filter yourself, so you can retain high measurement if needed. Set the maximum value to 1000 μg/m³, for instance, and look at the sensors one at a time.",
```

Comment:
spelling error in measurement, use 90-percentile

Reply:
adjusted